extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/StandardSecurityCheckInterceptor.java

```diff
@@ -58,6 +59,16 @@ public static final class RolesAllowedInterceptor extends StandardSecurityCheckI
 
     }
 
+    /**
+     * Prevent the SecurityHandler from performing {@link io.quarkus.security.PermissionsAllowed} security checks
```

Comment:
Oh yeah :-) Remember these bugs where there was issues with blocking when accessing identity on IO thread? There is `EagerSecurityHandler` in RR that performs security checks before serialization and therefore these is no point to perform same checks again. I think it is explain here https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/StandardSecurityCheckInterceptor.java#L24

I actually plan to check if CDI changes allows to avoid this approach as we discussed in the PR https://github.com/quarkusio/quarkus/issues/26536